ksqldb-serde/src/main/java/io/confluent/ksql/serde/connect/ConnectDataTranslator.java

```diff
@@ -41,8 +43,15 @@
 
   private final Schema schema;
 
+  private final Optional<Schema> phyiscalSchema;
+
   public ConnectDataTranslator(final Schema schema) {
+    this(schema, null);
+  }
+
+  public ConnectDataTranslator(final Schema schema, final Schema physicalSchema) {
     this.schema = Objects.requireNonNull(schema, "schema");
+    this.phyiscalSchema = Optional.ofNullable(physicalSchema);
```

Comment:
If null is accepted and the meaning is just that the parameter is optional, then let's pass `Optional` directly. It is a common practice we have followed in the ksql code.